### DISCUSSION TITLE
Process manager pimcore 11

### DISCUSSION THREAD

**Discussion by nishitsureka** [pimcore/pimcore#16033] [Q&A / Support]
Hi everyone,
I tried installing process manager in pimcore 11
But I am getting failed to execute command.
I dont know where I am going wrong or did I missed any configuration.

**Comment by nishitsureka** [comment]
hi everyone.
any solution

**Reply by dvesh3** [reply]
please check with command `php bin/console pimcore:bundle:list` if the bundle is properly enabled and installed.

**Reply by nishitsureka** [reply]
yes, It is enabled and installed. but whenever i tried to execute it is failed to execute.
i am trying on local window. is this the reason?

**Reply by dvesh3** [reply]
@ctippler could you please help?

**Comment by aryaantony92** [comment]
@nishitsureka Are you facing issue with executing the migration ? If so, could you please double check if the version of process-manager bundle is 5.0.4. https://github.com/pimcore/pimcore/discussions/15890#discussioncomment-6986487

**Reply by nishitsureka** [reply]
@aryaantony92 
"elements/process-manager-bundle": "^5.0",
